connectors/src/api/update_connector.ts

```diff
@@ -66,27 +56,10 @@ const _getConnectorUpdateAPIHandler = async (
     });
   }
 
-  const updateRes = await (async () => {
-    switch (provider2createConnectorType[connector.type]) {
-      case "oauth": {
-        const connectorUpdater = UPDATE_CONNECTOR_BY_TYPE[
-          connector.type
-        ] as ConnectorUpdaterOAuth;
-        const params = connectorParams as CreateConnectorOAuthRequestBody;
-
-        return connectorUpdater(connector.id, {
-          connectionId: params.connectionId,
-        });
-      }
-      case "url": {
-        const params = connectorParams as CreateConnectorUrlRequestBody;
-        const connectorUpdater = UPDATE_CONNECTOR_BY_TYPE[
-          connector.type
-        ] as ConnectorUpdaterUrl;
-        return connectorUpdater(connector.id, params);
-      }
-    }
-  })();
+  const connectorUpdater = UPDATE_CONNECTOR_BY_TYPE[connector.type];
```

Comment:
Completely removed everything related to configuration from the /update endpoint.